frontend/src/components/input.module.css

```diff
@@ -1,54 +1,142 @@
 .inputContainer {
   box-sizing: border-box;
-  display: inline-block;
-  margin: 1rem 0 2rem 0;
+  display: flex;
+  align-items: flex-start;
   position: relative;
   width: 50%;
 }
 
-.input {
-  background-color: var(--border-primary);
-  border-radius: 8px;
-  border: none;
+.textarea {
+  flex: 1;
+  border-radius: 24px;
+  border: 1px solid var(--grey-500);
   box-sizing: border-box;
-  color: var(--text-color-primary);
-  font-size: 1rem;
-  height: 3rem;
+  font-family: 'Roboto', sans-serif;
+  background-color: var(--grey-100);
+  color: var(--grey-900);
+  font-size: 16px;
   outline: none;
-  padding: 0 34px 0 10px;
-  width: 100%;
+  resize: none;
+  padding: 12px 76px 12px 24px;
+  min-height: 48px;
+  max-height: 110px;
+  overflow-y: hidden;
 }
 
-.input:focus {
-  border: 1px solid var(--blue);
+.textarea:focus {
+  background-color: var(--grey-50);
+  border: 2px solid var(--grey-800);
 }
 
-.sendButton {
+.textarea:hover {
+  background-color: var(--grey-200);
+}
+
+.textarea::-webkit-scrollbar {
```

Comment:
For the scroll bar theming there is also the [scrollbar-color](https://developer.mozilla.org/en-US/docs/Web/CSS/scrollbar-color) and [scrollbar-width](https://developer.mozilla.org/en-US/docs/Web/CSS/scrollbar-width) standard properties that will also support Firefox. However, that reduces the styling in webkit compared to these.